modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/order/test/CommerceOrderTest.java

```diff
@@ -140,6 +143,25 @@ public void tearDown() {
 		CentralizedThreadLocal.clearShortLivedThreadLocals();
 	}
 
+	@Test
+	public void testCommerceOrderCurrency() throws Exception {
+		CommerceCurrency commerceCurrency2 = CommerceCurrencyTestUtil.addCommerceCurrency(
+			_group.getCompanyId());
```

Comment:
@alecsloan This should just be named `commerceCurrency` because there is no `commerceCurrency1`.

Reply:
Hmm I'm not sure I've modified this test in my changes

Reply:
oh yeah sorry just noticed it was outdated.